### PR TITLE
Add: socks-proxy in infra

### PR DESCRIPTION
**Pull request by Fraggle**
## Description

Add the necessary service & app to deploy a socks proxy.

## Risk

I don't really know what I'm doing but @fontanierh told me it was safe.

## Deploy Plan

Apply infra.

### CODE REVIEW

File: k8s/services/socks-proxy-service.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: socks-proxy-service
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: "socks-proxy-static-ip"
+spec:
+  type: LoadBalancer
```
Comment (fontanierh) [inline k8s/services/socks-proxy-service.yaml:8]: still not sure I fully understand this ([here](https://cloud.google.com/kubernetes-engine/docs/concepts/service-load-balancer?hl=fr#load_balancer_types) they mention that a GKE annotation might be necessary) but down to try it 👍

**Review by fontanierh — APPROVED** [review]
🔥

File: k8s/services/socks-proxy-service.yaml (hunk 1 above)
Comment (Fraggle) [reply k8s/services/socks-proxy-service.yaml:9]: Me neither, added "ca mange pas de pain".

**Comment by spolu** [comment]
You probably already have this, this is a somewhat critical part of our infrastructure and that these pull requests are public for all our users to see and review.

**Pull request merged**